ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -132,12 +133,15 @@ def __init__(self):
         self.check_param('dut_username', '', required=True)
         self.check_param('dut_password', '', required=True)
         self.check_param('dut_hostname', '', required=True)
+        self.check_param('is_dualtor', False, required=False)
```

Comment:
Minor: Do you need this param to be sent? I think you could derive this using `self.vlan_mac != self.dut_mac`. You could store is_dualtor during init and reuse later.